### ISSUE TITLE
Requested shape requires a multiple of 64

### ISSUE THREAD

**Issue by tqshao** [tensorflow/models#1799]
I am training on my own dataset with faster rcnn resnet101 model.
After some steps I got this error:

`InvalidArgumentError (see above for traceback): Input to reshape is a tensor with 4092 values, but the requested shape requires a multiple of 64
	 [[Node: Loss/BoxClassifierLoss/Reshape_129 = Reshape[T=DT_FLOAT, Tshape=DT_INT32, _device="/job:localhost/replica:0/task:0/gpu:0"](Loss/BoxClassifierLoss/boolean_mask/Gather, Loss/BoxClassifierLoss/Reshape_129/shape)]]
	 [[Node: gradients/FirstStageFeatureExtractor/resnet_v1_101/resnet_v1_101/block3/unit_8/bottleneck_v1/conv3/convolution_grad/tuple/control_dependency_1/_15037 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/cpu:0", send_device="/job:localhost/replica:0/task:0/gpu:0", send_device_incarnation=1, tensor_name="edge_44394_gradients/FirstStageFeatureExtractor/resnet_v1_101/resnet_v1_101/block3/unit_8/bottleneck_v1/conv3/convolution_grad/tuple/control_dependency_1", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/cpu:0"]()]]
`
My config file:
model {
  faster_rcnn {
    num_classes: 23
    image_resizer {
    keep_aspect_ratio_resizer {
        min_dimension: 256
        max_dimension: 1024
      }
    fixed_shape_resizer {
        height: 224
        width: 224
      } 

    }
    feature_extractor {
      type: 'faster_rcnn_resnet101'
      first_stage_features_stride: 16
    }
    first_stage_anchor_generator {
      grid_anchor_generator {
        scales: [0.25, 0.5, 1.0, 2.0]
        aspect_ratios: [0.5, 1.0, 2.0]
        height_stride: 16
        width_stride: 16
      }
    }
    first_stage_box_predictor_conv_hyperparams {
      op: CONV
      regularizer {
        l2_regularizer {
          weight: 0.0
        }
      }
      initializer {
        truncated_normal_initializer {
          stddev: 0.01
        }
      }
    }
    first_stage_nms_score_threshold: 0.0
    first_stage_nms_iou_threshold: 0.7
    first_stage_max_proposals: 400
    first_stage_localization_loss_weight: 2.0
    first_stage_objectness_loss_weight: 1.0
    initial_crop_size: 14
    maxpool_kernel_size: 2
    maxpool_stride: 2
    second_stage_box_predictor {
      mask_rcnn_box_predictor {
        use_dropout: false
        dropout_keep_probability: 1.0
        fc_hyperparams {
          op: FC
          regularizer {
            l2_regularizer {
              weight: 0.0
            }
          }
          initializer {
            variance_scaling_initializer {
              factor: 1.0
              uniform: true
              mode: FAN_AVG
            }
          }
        }
      }
    }
    second_stage_post_processing {
      batch_non_max_suppression {
        score_threshold: 0.0
        iou_threshold: 0.6
        max_detections_per_class: 100
        max_total_detections: 300
      }
      score_converter: SOFTMAX
    }
    second_stage_localization_loss_weight: 2.0
    second_stage_classification_loss_weight: 1.0
  }
}

train_config: {
  batch_size: 16
  optimizer {
    momentum_optimizer: {
      learning_rate: {
        manual_step_learning_rate {
          initial_learning_rate: 0.0001
          schedule {
            step: 0
            learning_rate: .0001
          }
          schedule {
            step: 40000
            learning_rate: .00001
          }
          schedule {
            step: 100000
            learning_rate: .000001
          }
        }
      }
      momentum_optimizer_value: 0.9
    }
    use_moving_average: false
  }
  gradient_clipping_by_norm: 10.0
  fine_tune_checkpoint: "models/object_detection/data/model/model.ckpt-37093"
  from_detection_checkpoint: true
  num_steps: 800000
  data_augmentation_options {
    random_horizontal_flip {
    }
  }
}

train_input_reader: {
  tf_record_input_reader {
    input_path: "object_detection/data/all_train.record"
  }
  label_map_path: "object_detection/data/all.pbtxt"
}

eval_config: {
  num_examples: 4952
}

eval_input_reader: {
  tf_record_input_reader {
    input_path: "object_detetction/data/pascal_val.record"
  }
  label_map_path: "object_detectiondata/all.pbtxt"
}

Even though I resized all the input images to the same size, it still occurs? Do you have any idea?

**Comment by aselle** [comment]
I apologize but I am having a hard time understanding what the problem is, where the problem is, and what version it affects. Please resubmit and pay attention to the issue template (https://github.com/tensorflow/tensorflow/issues/new) . Please provide all the information it asks. Thank you.
